src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionCategoryServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package pl.byczazagroda.trackexpensesappbackend.service;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionCategoryDTO;
+import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionCategoryDetailedDTO;
+import pl.byczazagroda.trackexpensesappbackend.exception.AppRuntimeException;
+import pl.byczazagroda.trackexpensesappbackend.exception.ErrorCode;
+import pl.byczazagroda.trackexpensesappbackend.mapper.FinancialTransactionCategoryModelMapper;
+import pl.byczazagroda.trackexpensesappbackend.model.FinancialTransactionCategory;
+import pl.byczazagroda.trackexpensesappbackend.repository.FinancialTransactionCategoryRepository;
+import pl.byczazagroda.trackexpensesappbackend.repository.FinancialTransactionRepository;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import java.math.BigInteger;
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+@Validated
+public class FinancialTransactionCategoryServiceImpl implements FinancialTransactionCategoryService {
+
+    private final FinancialTransactionCategoryRepository financialTransactionCategoryRepository;
```

Comment:
Insert one blank line below, please. Mean there should be an empty line:

between every field in the class,
between every method in the class,
import and class declaration
between the last method declaration character } and the end of the class declaration character }

Reply:
done, thank you